Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "JuliaInterpreter"
 uuid = "aa1ae85d-cabe-5617-a682-6adf51b2e16a"
-version = "0.3.2"
+version = "0.3.3"
```

Comment:
I wonder if we should go to 0.4? "frames now keep their callees when the interpreter throws an error" might be technically breaking. I doubt anyone is relying on the current behavior :smile:, but it's also just a giant change in internal behavior?

Anyway, I'm fine with either, and definitely approve of a new release.

Reply:
0.4.0 it is!